prepay-cod.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
```

Comment:
Why can't we merge this in `api/prepay-cod.php`?

Reply:
moved it